bin/composer-post-install-script.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+// phpcs:disable PSR1.Files.SideEffects.FoundWithSymbols
+
+function copyFile(array $file): void
+{
+    if (is_readable($file['destination'])) {
+        echo "File {$file['destination']} already exists." . PHP_EOL;
+    } else {
+        if (! copy($file['source'], $file['destination'])) {
+            echo "Cannot copy {$file['source']} file to {$file['destination']}" . PHP_EOL;
+        } else {
+            echo "File {$file['source']} copied successfully to {$file['destination']}." . PHP_EOL;
+        }
+    }
+}
+
+$files = [
+    [
+        'source'      => 'config/autoload/local.php.dist',
+        'destination' => 'config/autoload/local.php',
+    ],
+    [
+        'source'      => 'config/autoload/local.test.php.dist',
+        'destination' => 'config/autoload/local.test.php',
+    ],
+    [
+        'source'      => 'vendor/dotkernel/dot-mail/config/mail.global.php.dist',
```

Comment:
```suggestion
        'source'      => 'config/autoload/mail.global.php.dist',
```
`mail.global.php.dist` already created at the previous iteration, you can use that as the source file for this operation.

Reply:
I think it's better to keep the same source, because copying files could fail, and you have to make sure the files are copied in a specific order (.dist files first).

Reply:
👍